### PR TITLE
server: stop caching session.js, add more flexible caching, stop using tape interpolation

### PR DESCRIPTION
**Pull request by tacryt-socryp**
https://github.com/urbit/landscape/issues/561

### CODE REVIEW

**Comment by matildepark** [comment]
I think this could target next-userspace fine enough, it's a nice improvement.

File: pkg/arvo/app/file-server.hoon [hunk 1]
```diff
@@ -188,8 +188,11 @@
     ?:  ?=([%'~landscape' %js %session ~] site.req-line)
       %+  require-authorization-simple:app
         inbound-request
-      %-  js-response:gen
-      (as-octt:mimes:html "window.ship = '{+:(scow %p our.bowl)}';")
+      %.  %-  as-octs:mimes:html
+          :((cury cat 3) 'window.ship = "' (rsh 3 (scot %p our.bowl)) '";')
```
Comment (joemfb) [inline pkg/arvo/app/file-server.hoon:192]: `(rap 3 'window.ship = "' (rsh 3 (scot %p our.bowl)) '";' ~)` is preferable, as it's a single jetted call (and single allocation). (This specific case might be too small to matter, but it's worth establishing the habit.)

**Comment by tacryt-socryp** [comment]
@liam-fitzgerald ready to go

**Comment by tacryt-socryp** [comment]
@liam-fitzgerald please review this

**Review by liam-fitzgerald — APPROVED** [review]

**Pull request merged**